Jenkinsfile

```diff
@@ -17,6 +17,7 @@
 // Then a second PR submitted to comment out the @Library line, and when it
 // is landed, both PR branches can be deleted.
 //@Library(value="trusted-pipeline-lib@my_pr_branch") _
+@Library(value="trusted-pipeline-lib@sre-3030") _
```

Comment:
Doesn't this need to be removed before landing?
```suggestion
```

Reply:
Yes.
Depends what procedure we undertake.
The documentation says: "2 PRs" , but I think one is enough.